### PR TITLE
Bug 1139963 - ensuring that action menu and activity window are exclusiv...

### PR DESCRIPTION
**Pull request by yzen**
...ely visible to the screen reader.

### CODE REVIEW

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=cd12f34f024f)

File: apps/system/js/app_window_manager.js [hunk 1]
```diff
@@ -599,10 +607,24 @@
           activeApp && activeApp.broadcast('hidewindow', evt.detail);
           break;
 
+        case 'activitycreated':
```
Comment (alivedise) [inline apps/system/js/app_window_manager.js]: Don't do this here. Do it in ChildWindowFactory.

File: apps/system/js/app_window_manager.js [hunk 2]
```diff
@@ -599,10 +607,24 @@
...
+          // Set screen reader visibility to false for active app's browser's
+          // element only.
+          activeApp._setVisibleForScreenReader(false);
+          break;
+
+        case 'activityterminated':
```
Comment (alivedise) [inline apps/system/js/app_window_manager.js]: Same.

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7fb458385103)

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a7efae1e72ab)

**Pull request merged**